src/scripts/node-group-rollover.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Check if eksctl and aws are installed
+command -v eksctl >/dev/null 2>&1 || { echo >&2 "eksctl is required but not installed. Aborting."; exit 1; }
```

Comment:
Perhaps this part ought to be put in a separate script, like check-tools.sh, since you are doing similar checks in 3 shell scripts?